lib/capnp_c.h

```diff
@@ -329,10 +329,12 @@ CAPN_INLINE int capn_write1(capn_ptr p, int off, int val) {
 	if (off >= p.datasz*8) {
 		return -1;
 	} else if (val) {
-		((uint8_t*)p.data)[off/8] |= (uint8_t)(1 << (off%8));
+		uint8_t tmp = (uint8_t)(1 << ( off & 7));
+		( (uint8_t*)p.data )[off>>3] |= tmp;
```

Comment:
.../capnp_c.h:352:29: error: conversion to ‘uint8_t {aka unsigned char}’ from ‘int’ may alter its value [-Werror=conversion]
   ((uint8_t*)p.data)[off/8] |= (uint8_t)(1 << (off%8));
                             ^
.../capnp_c.h:357:29: error: conversion to ‘uint8_t {aka unsigned char}’ from ‘int’ may alter its value [-Werror=conversion]
   ((uint8_t*)p.data)[off/8] &= (uint8_t)~(1 << (off%8));

under gcc/g++ (Ubuntu 4.9.2-10ubuntu13) 4.9.2